kornia/geometry/conversions.py

```diff
@@ -1175,3 +1185,163 @@ def normalize_homography3d(
+        extrinsics: pose matrix :math:`(B, 4, 4)`.
+
+    Returns:
+        extrinsics: pose matrix :math:`(B, 4, 4)`.
+    """
+    KORNIA_CHECK_SHAPE(extrinsics_graphics, ["B", "4", "4"])
+    invert_yz = tensor(
+        [[[1, 0, 0, 0], [0, -1, 0, 0], [0, 0, -1, 0], [0, 0, 0, 1.0]]],
+        dtype=extrinsics_graphics.dtype,
+        device=extrinsics_graphics.device,
+    )
+    return extrinsics_graphics @ invert_yz
+
+
+def camtoworld_vision_to_graphics(extrinsics_vision: Tensor) -> Tensor:
+    r"""Converts vision coordinate frame (e.g. OpenCV) to graphics coordinate
+    frame (e.g. OpenGK.).
+    Graphics convention: [+x, +y, +z] == [right, up, backwards].
+    Vision convention: [+x, +y, +z] == [right, down, forwards]
+
+    Args:
+        extrinsics: pose matrix :math:`(B, 4, 4)`.
+
+    Returns:
+        extrinsics: pose matrix :math:`(B, 4, 4)`.
+    """
+    KORNIA_CHECK_SHAPE(extrinsics_vision, ["B", "4", "4"])
+    invert_yz = torch.tensor(
+        [[[1, 0, 0, 0], [0, -1, 0, 0], [0, 0, -1, 0], [0, 0, 0, 1.0]]],
+        dtype=extrinsics_vision.dtype,
+        device=extrinsics_vision.device,
+    )
+    return extrinsics_vision @ invert_yz
+
+
+def poseRt_to_camtoworldRt(R: Tensor, t: Tensor) -> Tuple[Tensor, Tensor]:
+    r"""Converts camera pose in world coordinates to camtoworld frame,
+    e.g. used in Colmap
+
+    Args:
+        R: Rotation matrix, :math:`(B, 3, 3).`
+        t: Translation matrix :math:`(B, 3, 1)`.
+
+    Returns:
+        Rinv: Rotation matrix, :math:`(B, 3, 3).`
+        tinv: Translation matrix :math:`(B, 3, 1)`.
+    """
+    KORNIA_CHECK_SHAPE(R, ["B", "3", "3"])
+    KORNIA_CHECK_SHAPE(t, ["B", "3", "1"])
+
+    R_inv = R.transpose(1, 2)
+
+    new_t: Tensor = -R_inv @ t
+
+    return (R_inv, new_t)
+
+
+def camtoworldRt_to_poseRt(R: Tensor, t: Tensor) -> Tuple[Tensor, Tensor]:
+    r"""Converts camtoworld frame e.g. used in Colmap to pose.
+
+    Args:
+        R: Rotation matrix, :math:`(B, 3, 3).`
+        t: Translation matrix :math:`(B, 3, 1)`.
+
+    Returns:
+        R: Rotation matrix, :math:`(B, 3, 3).`
+        t: Translation matrix :math:`(B, 3, 1)`.
+    """
+    KORNIA_CHECK_SHAPE(R, ["B", "3", "3"])
+    KORNIA_CHECK_SHAPE(t, ["B", "3", "1"])
+
+    R_inv = R.transpose(1, 2)
+
+    new_t: Tensor = -R_inv @ t
+
+    return (R_inv, new_t)
+
+
+def screenpose_to_camerapose(extrinsics_screen: Tensor) -> Tensor:
+    r"""Converts pose of the phonescreen to pose of the camera.
+    See https://developer.apple.com/documentation/arkit/arconfiguration/worldalignment/gravity
+
+    Camera convention: [+x, +y, +z] == [right, down, forwards]
+    Screen convention: [+x, +y, +z] == [right, up, backwards]
+
+    Args:
+        extrinsics: pose matrix :math:`(B, 4, 4)`.
+
+    Returns:
+        extrinsics: pose matrix :math:`(B, 4, 4)`.
+    """
+    return camtoworld_vision_to_graphics(extrinsics_screen)
+
+
+def ARKitQTVecs_to_ColmapQTVecs(qvec: Tensor, tvec: Tensor) -> Tuple[Tensor, Tensor]:
```

Comment:
add todo here to integrate new upcoming quaternion class API https://github.com/kornia/kornia/pull/1801